### DISCUSSION TITLE
How would I have a quest completion be based on an item tag?

### DISCUSSION THREAD

**Discussion by ghost** [FTBTeam/FTB-Mods-Issues#1147] [Q&A] [answered]
Right now, you're only able to add items specifically as a quest completion requirement, but I also want to use item tags instead such as `#minecraftcraft:logs`

**Comment by desht** [comment, marked as answer]
Ensure you have FTB XMod Compat installed, and also install one or both of Item Filters and FTB Filter System.

Both Item Filters and FFS support tag filters, matching items by tag.  In addition (when you have either or both filter mods installed), FTB Quests supports converting items in item tasks to item tags: right-click on the item task in the quest view panel, and select "Convert to Item Tag".
